conf/carbon.conf.example

```diff
@@ -347,6 +353,11 @@ REPLICATION_FACTOR = 1
 # must be defined in this list
 DESTINATIONS = 127.0.0.1:2004
 
+# This define the protocol to use to contact the destination. It can be
+# set to one of "line", "pickle", "udp" and "protobuf". This list can be
+# extended with CarbonClientFactory plugins and defaults to "pickle".
+# DESTINATION_PROTOCOL = pickle
```

Comment:
DESTINATION_PROTOCOL comes from https://github.com/graphite-project/carbon/blob/0a6944d144955dc398f722384bce2e42ed348f61/lib/carbon/client.py#L379

It's used the control the protocol between carbon instances, which by default is pickle but has been made configurable since the introduction of plugable protocols.